javascript/app.js

```diff
@@ -152,12 +161,20 @@ cocktailApp.showDrinkPreview = function(drink) {
         i: cocktailApp.currentDrink
     });
 }
-// method to take user input and find a random drink:
+// method to get random drink:
```

Comment:
method to get random drink since this was done in 3 cases on the submit button - returns the random drink so it can be used as an argument in a function